service/dap/daptest/client.go

```diff
@@ -150,7 +150,7 @@ const ProcessExited = `Process [0-9]+ has exited with status %s\n`
 
 func (c *Client) ExpectOutputEventProcessExited(t *testing.T, status int) *dap.OutputEvent {
 	t.Helper()
-	return c.ExpectOutputEventRegex(t, fmt.Sprintf(ProcessExited, fmt.Sprintf("%d", status)))
+	return c.ExpectOutputEventRegex(t, fmt.Sprintf(ProcessExited, fmt.Sprintf("(%d|-1)", status)))
```

Comment:
Pease add a comment here why it is needed

Reply:
Done.